external/grpc/README.md

```diff
@@ -0,0 +1,110 @@
+# gRPC framework for TizenRT.
+TizenRT features the gRPC framework from the opensource project [gRPC](https://github.com/grpc/grpc), and is versioned at 1.9.
+TizenRT's version of gRPC supports only C++ binding, so it is expected that the developer should write C++ applications in order to 
```

Comment:
As you wrote, let's change a line for readablility. Add 2 spaces at end of previous line.